curriculum/challenges/english/25-front-end-development/workshop-music-player/6751d5d32e58d065652e26a5.md

```diff
@@ -0,0 +1,733 @@
+---
+id: 6751d5d32e58d065652e26a5
+title: Step 36
+challengeType: 0
+dashedName: step-36
+---
+
+# --description--
+
+Loop through the `playlistSongElements` and use the `removeAttribute()` method to remove the `"aria-current"` attribute for each of the songs.
+
+Then, create an `if` statement with the condition `songToHighlight` to add the attribute back to the currently playing song. Within the `if` statement use `setAttribute` on `songToHighlight` and pass in `"aria-current"` and `"true"` as the first and second arguments.
```

Comment:
Why can't I...
```js
const highlightCurrentSong = () => {
  const currents = document.querySelectorAll('.playlist-song[aria-current="true"]');
  currents.forEach(e => e.removeAttribute("aria-current"));
  const songToHighlight = document.getElementById(
    `song-${userData.currentSong?.id}`
  );
  songToHighlight.setAttribute('aria-current', "true")
};
```

Reply:
```js
const highlightCurrentSong = () => {
  const previousCurrentSong = document.querySelector('.playlist-song[aria-current="true"]');
  if (previousCurrentSong) previousCurrentSong.removeAttribute("aria-current");

  const songToHighlight = document.getElementById(
    `song-${userData.currentSong?.id}`
  );

  if (songToHighlight) songToHighlight.setAttribute("aria-current", "true");
};
```
Would that be okay?

Reply:
if there can be only one `.playlist-song[aria-current="true"]`, yes. If there can be multiple this could cause a small bug